libs/wingc/src/jsify.rs

```diff
@@ -307,30 +307,6 @@ impl<'a> JSifier<'a> {
 		}
 	}
 
-	// This helper determines what requirement and dependency to add to the struct schema based
-	// on the type annotation of the field.
-	// I.E. if a struct has a field named "foo" with a type "OtherStruct", then we want to add
-	// the field "foo" as a required  and the struct "OtherStruct" as a dependency. so the result is
-	// a tuple (required, dependency)
-	fn extract_struct_field_schema_dependency(
-		&self,
-		typ: &TypeAnnotationKind,
-		field_name: &String,
-	) -> (Option<String>, Option<String>) {
```

Comment:
I moved this to be a free floating function since it doesn't need/use the `&self` parameter